custom_components/webhook_service/__init__.py

```diff
@@ -1,27 +1,28 @@
-import logging
-import requests 
-import json
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
 
-DOMAIN = "webhook_service"
-_LOGGER = logging.getLogger(__name__)
+from .const import DOMAIN, WEBHOOKS_DATAS
 
+import logging
+_LOGGER = logging.getLogger(__name__)
 
-SERVICE_SEND = "basic_webhook"
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the integration."""
+    if DOMAIN not in hass.data:
+        hass.data[DOMAIN] = {}
+        await _setup_webhooks(hass, config)
+    return True
 
-def setup(hass, config):
-    def send_basic_webhook(call):
-        data = call.data.copy()
-        # if exists json, send json string
-        if "json" in data:
-            jsondata = json.loads(data["json"])
-        # if exists jsonObj, stringify jsonObj then send it
-        elif "jsonObj" in data:
-            jsondata = data["jsonObj"]
-        else:
```

Comment:
A prior merge had added the functionality for jsonObj, it would be good if this could be maintained.

Reply:
Yeah I most likely have forgot about it, will add ass soon as possible.